README.md

```diff
@@ -49,6 +49,7 @@ paginate = 4
 [params]
   author = "Hugo de Groot"
   siteHeading = "Hugo Grotius" # defaults to author
+  # favicon = "favicon-gnu.png" # add favorite/URL/tab/shortcut icon
```

Comment:
```suggestion
  # favicon = "favicon-gnu.png" # Adds a small icon next to the page title in a tab
```

I guess your explanation is a more encompassing, but I'd like it a bit more descriptive for those who don't know what a favicon is. This should also be changed in the config.toml in that case.